### ISSUE TITLE
Javascript functions are not called

### ISSUE THREAD

**Issue by r-shayestehpour** [adamhartford/SwiftR#18]
I call the following method in `viewWillAppear` method:

    func startSignalRConnection() {
        SwiftR.connect(self.pusherURL) { connection in
            print("connection ID: ", connection.connectionID)
        }
    }

but the app crashes before printing anything here, because in the method `shouldHandleRequest(request: NSURLRequest) -> Bool` the variable `msg` is an empty string and it cannot parse it as json. In order to diagnose the problem, I edited the method like this:
    
    func shouldHandleRequest(request: NSURLRequest) -> Bool {
        if request.URL!.absoluteString.hasPrefix("swiftr://") {
            print(request.URL!.absoluteString)
            let id = (request.URL!.absoluteString as NSString).substringFromIndex(9)
            let msg = webView.stringByEvaluatingJavaScriptFromString("readMessage(\(id))")!
            let data = msg.dataUsingEncoding(NSUTF8StringEncoding, allowLossyConversion: false)!
            
            //debug
            
            let msg2 = webView.stringByEvaluatingJavaScriptFromString("console.log(swiftR.foobar)")!

            print("here1")
            print(id)
            print("here2")
            print(msg2)
            print("here3")
            print(msg)
            
            //debug
            
            let json: AnyObject = try! NSJSONSerialization.JSONObjectWithData(data, options: [])
            
            processMessage(json)

            return false
        }
        
        return true
    }

`swiftR.foobar` is a variable I declared in the `SwiftR.js` file:

    window.swiftR = {
        connection: null,
        hubs: {},
        transport: 'auto',
        headers: {},
        foobar : 1234
    };

the output to those logs is as follows:

    here1
    91bihm8e
    here2
    //empty line(empty string)
    here3
    //empty line

Is there something wrong with the way I'm calling the functions?

**Comment by adamhartford** [comment]
Looks like there's a bug in the UIWebView branch of the message handling code after a recent change:

```swift
let msg = webView.stringByEvaluatingJavaScriptFromString("readMessage(\(id))")!
```

should be:

```swift
let msg = webView.stringByEvaluatingJavaScriptFromString("readMessage('\(id)')")!
```

You make that change in your copy of the code, or use WKWebView instead of UIWebView (if it's appropriate for your use case):

```swift
// See the demo project
SwiftR.useWKWebView = true
```

Either way, I will update the pod today with this fix. Thanks for submitting this issue.

**Comment by adamhartford** [comment]
Fixed in v0.8.3.
